lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -613,12 +613,12 @@ def test_single_problem(self, _get_current_task):
             metadata={'graded': True},
             display_name='Problem Vertical'
         )
-        self.define_option_problem(u'Pröblem1', parent=vertical)
```

Comment:
Per our discussion I am going to revert this and try to figure out the URL issue.